src/tito/release/distgit.py

```diff
@@ -343,7 +346,7 @@ def _git_upload_sources(self, project_checkout):
         output = run_command(cmd)
         debug(output)
         debug("Removing write-only permission on:")
-        for filename in self.builder.sources:
+        for filename in sources:
             run_command("chmod u+w %s" % filename)
```

Comment:
off-topic: is the "removing write-only permission" a correct debug entry?  Seems like we are actually adding write permission to the user.

Reply:
You are right, fixed.